src/components/WebCLI/WebCLI.tsx

```diff
@@ -222,7 +224,11 @@ const WebCLI = ({
     }
   };
 
-  const showHelp = () => {
+  const showHelp = (event: React.MouseEvent | React.KeyboardEvent) => {
+    // Only trigger the help from the space or enter keys (or mouse clicks).
```

Comment:
I initially thought this was related, so I fixed it and left the change in as we need it anyway.